.teamcity/settings.kts

```diff
@@ -168,23 +173,25 @@ project {
             params {
                 select(
                     "releaseScope", "", label = "releaseScope", description = "The scope of the release",
-                    display = ParameterDisplay.PROMPT, options = listOf("major", "minor", "patch")
+                    display = PROMPT, options = listOf("major", "minor", "patch")
                 )
                 text(
-                    "githubUsername",
+                    "env.GRGIT_USER",
```

Comment:
❓ Can you explain to me what this is supposed to do? Are those `GRGIT_*` internal (undocumented?) environment variables?

Reply:
nebula plugin (which we use directly) uses grgit plugin which then tags Git and needs these credentials - see http://ajoberstar.org/grgit/grgit-authentication.html